_data/community_neurons.yml

```diff
@@ -212,6 +218,12 @@
   logo: translate.png
   demos:
 
+- neuron_name: TwitchTv
+  description: Ask Kalliope who of your favorite players are streaming now.
+  doc_url: https:://github.com/bacardi55/kalliope-twitchtv
```

Comment:
typo in the doc_url (double colon) : https:://github.com/bacardi55/kalliope-twitchtv